plugin.video.fantasticc/default.py

```diff
@@ -344,19 +344,28 @@ def INDEXCOLLECT(url):   # Index Collections Pages
     xbmc.log('URL Loading: %s' % url)
     html = get_html(url)
 
-    match = re.compile(
-        '<a class="clnk" href="([^"]+)">([^<]+)</a>.+?"counter-right">'
-        '(\d+) videos.+?style="background: url\(([^)]+)\);',
-        re.DOTALL).findall(html)
-    for gurl, name, num_of_vids, icons in match:
+    match = re.compile('<a class="clnk" href="(.+?)">(.+?)</a>(.+?)<div class="tag-list">', re.DOTALL).findall(html)
+
+    for gurl, name, chtml in match:
         xbmc.log('Name [%s]' % name)
         realurl = 'http://fantasti.cc%s' % gurl
         name = unescape(name)
         mode = 1
-        if num_of_vids == str(0):
-            continue
 
-        addDir(name + ' (' + num_of_vids + ' vids)', realurl, mode, icons)
+        #scrape number of vids
+        vidnumber = re.search('"videosListNumber"><b>(.*?)<',chtml)
```

Comment:
re.search is intentionally used as find all would result in "None" in case of videos<5
by using re.search, it will not return None and code is cleaner.
Let me know if it is not acceptable and I will resubmit with both 80 char and findall corrections